executor/executor.yml

```diff
@@ -33,6 +33,28 @@
 - set_fact:
     envNameList: ["APP_NAMESPACE", "APP_LABEL", "APP_KIND", "CHAOSENGINE"]
     envValueList: ["{{ c_app_ns }}", "{{ c_app_label }}", "{{ c_app_kind }}", "{{ c_engine }}"]
+    c_env_list: ""
+
+- name: Obtain the labels from the experiment
+  shell: >
+    kubectl get chaosexperiment -n {{ c_app_ns }} -o jsonpath='{.items[?(@.metadata.name=="{{ c_experiment_name }}")].spec.definition.labels}' | sed 's/map//g' | tr -d [ | tr -d ] | tr ":" =
+  args:
+    executable: /bin/bash
+  register: c_job_labels
+
+- name: Obtain the args from the experiment
+  shell: >
+    kubectl get chaosexperiment -n {{ c_app_ns }} -o jsonpath='{.items[?(@.metadata.name=="{{ c_experiment_name }}")].spec.definition.args[0]}{" \""}{.items[?(@.metadata.name=="{{ c_experiment_name }}")].spec.definition.args[1]}{"\""}'
+  args:
+    executable: /bin/bash
+  register: c_job_args
+
+- name: Fetching data for the configmap
```

Comment:
Can we mention current limitations in a separate README ? For ex: ability to parse beyond 1 configmap etc., So that we keep track of things to do